pages/common/curl.md

```diff
@@ -5,32 +5,40 @@
 
 - Download a URL to a file:
 
-`curl "{{URL}}" -o {{filename}}`
+`curl {{http://example.com}} -o {{filename}}`
+
+- Download file using remote name:
```

Comment:
This description can be improved a bit. How about - "Download the contents to a file where the filename is extracted from the last part of the url" ?